examples/flutter_gallery/lib/demo/material/dialog_demo.dart

```diff
@@ -123,9 +128,14 @@ class DialogDemoState extends State<DialogDemo> {
                 context: context,
                 child: new AlertDialog(
                   title: const Text('Use Google\'s location service?'),
-                  content: new Text(
-                    _alertWithTitleText,
-                    style: dialogTextStyle
+                  content: new Semantics(
```

Comment:
seems like a place to use RouteName

Reply:
reverted for now, label is inferred from title